content/hi/docs/tasks/tools/included/verify-kubectl.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "kubectl इंस्टॉल सत्यापित करें"
+description: "kubectl कैसे सत्यापित करें."
+headless: true
+---
+
+kubectl को कुबेरनेट्स क्लस्टर को खोजने और एक्सेस करने के लिए, उसे
+[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) की आवश्यकता होती है,
+जो स्वचालित रूप से तब बनता है जब आप
+[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) का उपयोग करके क्लस्टर बनाते हैं
+या मिनीक्यूब क्लस्टर को सफलतापूर्वक डेप्लॉय करते हैं.
+डिफ़ॉल्ट रूप से, kubectl कॉन्फ़िगरेशन `~/.kube/config` पर स्थित होता है.
+
+जाँच करें कि क्लस्टर स्टेट प्राप्त करके kubectl ठीक से कॉन्फ़िगर किया गया है:
+
+```shell
+kubectl cluster-info
+```
+
+यदि आप एक URL प्रतिक्रिया देखते हैं, तो kubectl आपके क्लस्टर को एक्सेस करने के लिए सही ढंग से कॉन्फ़िगर किया गया है.
+
+यदि आपको निम्न जैसा संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं किया गया है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है.
```

Comment:
```suggestion
यदि आपको निम्न जैसा संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं हुआ है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है| 
```

Reply:
please review updated one

Reply:
I'm unable to commit these suggested changes as you are reviewing older commit.